From b477ef802d0112cd6dfa91117146552ac1bfdc81 Mon Sep 17 00:00:00 2001
From: Jacky Chou <jacky_chou@aspeedtech.com>
Date: Tue, 5 May 2026 13:32:40 +0800
Subject: [PATCH] reset: aspeed: extend AST2700 reset driver to cover AST2705

AST2705 shares the AST2700 reset controller, so extend the
RESET_AST2700 default condition to include CONFIG_ASPEED_AST2705.

Upstream-Status: Backport [SDK v11.03 migration]
Signed-off-by: Jacky Chou <jacky_chou@aspeedtech.com>
Change-Id: Iec82a6fffdc95171d514d27c409dbaf29c8ae680
Signed-off-by: rupeshs <rupeshs@ami.com>
---
 drivers/reset/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/reset/Kconfig b/drivers/reset/Kconfig
index 3642b0bc5f..2b5ae5deca 100644
--- a/drivers/reset/Kconfig
+++ b/drivers/reset/Kconfig
@@ -93,7 +93,7 @@ config RESET_AST2600
 config RESET_AST2700
 	bool "Reset controller driver for AST2700 SoCs"
 	depends on DM_RESET
-	default y if ASPEED_AST2700
+	default y if ASPEED_AST2700 || ASPEED_AST2705
 	help
 	  Support for reset controller on AST2700 SoC.
 	  Say Y if you want to control reset signals of different peripherals
-- 
2.43.0

